### PR TITLE
Add DTL logo

### PR DESCRIPTION
**Pull request by c-martinez**

### CODE REVIEW

**Comment by jspaaks** [comment]
Fine to merge, but DTL_short.jpg seems more suited for how we display it.

**Comment by c-martinez** [comment]
@mkuzak is supposed to have a look at the logo (I don't think I can add him as reviewer because he is not a collaborator in the repo?)

**Comment by jspaaks** [comment]
He is now!

**Comment by mkuzak** [comment]
The logo is really tiny. Maybe it can be cropped on the sides to make it larger when embedded in the square.

**Comment by jspaaks** [comment]
No that is not the idea of https://www.dtls.nl/about/dtl-logo/
I propose to use one of the other logos, specifically ``DTL_short.jpeg``

**Comment by mkuzak** [comment]
what is not the idea?
short version should be fine too.

**Comment by jspaaks** [comment]
Editing/cropping logos is not the idea, since that tends to upset organizations, especially those who have gone through the trouble of making several types of logos available through their website.

**Comment by c-martinez** [comment]
Changed logo -- could you check again?

File: static/json/pages.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
                     {
                       "title" : "DTL",
                       "link" : "https://www.dtls.nl/",
-                      "image" : "/img/DTL_full.jpg"
+                      "image" : "/img/DTL_short.jpg"
```
Comment (jspaaks) [inline static/json/pages.json]: jpeg not jpg

Comment (c-martinez) [reply static/json/pages.json]: Well spotted...

**Comment by mkuzak** [comment]
thank you, looks good to me

**Pull request merged**